vpr/src/base/place_and_route.cpp

```diff
@@ -355,7 +355,7 @@ int binary_search_place_and_route(const Netlist<>& placement_net_list,
                     auto& cluster_ctx = g_vpr_ctx.clustering();
                     // Cluster-based net_list is used for placement
                     print_place(filename_opts.NetFile.c_str(), cluster_ctx.clb_nlist.netlist_id().c_str(),
-                                filename_opts.PlaceFile.c_str());
+                                filename_opts.PlaceFile.c_str(), false);
```

Comment:
The false should be gone ...  just want print_place. No need for it to know if it is the initial placement or not.